com/course/zenkouv/src/test/java/webDriver/util/filter/OfferFilter.java

```diff
@@ -0,0 +1,21 @@
+package webDriver.util.filter;
+
+import webDriver.constant.OnlinerLocators;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import webDriver.util.validator.ElementValidator;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OfferFilter {
+
```

Comment:
OfferFilter and ProductFilter almost the same 
create `ElementFilter` with two param `List<WebElement>` and `By  by`

Reply:
fix